JASP-Engine/JASP/R/LSbinomialcommon.R

```diff
@@ -472,6 +474,20 @@
     }
   }
 }
+.sdBinomLS            <- function(N, p, prop = FALSE){
```

Comment:
I have no idea what this does.

Reply:
computes a standard deviation - sd - for a Binomial - Binom - distribution and it's a function from Learn Bayes - LS

Reply:
Then it's better to change the name of the function to reflect this.

Reply:
then give me your suggestion please because I have honestly no idea how to better reflect this: `base::sd` computes standard deviation, `Binom` - the name of distribution for which sd is computed, and it's camelCased

Reply:
.computeSdBinomLS

Reply:
changed